### ISSUE TITLE
Trying to render md-files outside of docs folder

### ISSUE THREAD

**Issue by awallemo** [mkdocs/mkdocs#2831]
I am trying to render md files that are located in the root directory of where the docs file is located. The build is run succesfully but I get a 404 not found error on the pages where I locate outside of docs. Can someone help me?

```
nav:
  - Get started:
    - Introduction:                         /../README.md                       
  - Documentation:
    - Creating a new report:             somefile.md
    - Roles and Access:                     someotherfile.md
  - Disaster Recovery:
    - Disaster Recovery Plan:               /../someotherdirectory/file.md
```

**Comment by pawamoy** [comment]
Try the snippets extension: https://facelessuser.github.io/pymdown-extensions/extensions/snippets/#snippets-notation

**Comment by awallemo** [comment]
@pawamoy not sure how this would help? I just want to use md files outside of the docs folder within my mkdocs-site. Is this possible?

**Comment by pawamoy** [comment]
Yes, that's what the snippets extension allows to do (among other things). I've found it to be the most reliable way to include files that are outside of the docs directory. For example, my `index.md` file has the contents of my `README.md``--8<-- "README.md"` (the readme is located one directory above the docs dir, but by default snippets use the `.` as base path)

**Comment by awallemo** [comment]
Ahh, I see, but its a python extension, and Im not using Python??

**Comment by pawamoy** [comment]
You definitely are using Python since MkDocs is written in Python :wink:

**Comment by awallemo** [comment]
I'm familiar with npm and understand that you need to download in a directory where package.json is located, but I dont have any experience with pip

**Comment by pawamoy** [comment]
You need to learn about pip and how to use it. If you installed MkDocs with another package manager (for example your OS package manager), then you should try to install pymdownx using this same package manager. MkDocs bugtracker is not the right place for this unfortunately. But there are *plenty* resources on the internet, I'm sure you'll quickly find how to handle this.

**Comment by awallemo** [comment]
Thank you for your help :)

**Comment by awallemo** [comment]
Okay, I've now got the extension I think. If I understand the example correctly. If I have a root directory and within this root directory I have my docs folder which mkdocs reads from. If I wanted the introduction navigation to point to the README.md which is also located in the root directory (not within docs folder), would I do it like this? 

```
nav:
  - Get started:
    - Introduction:                         --8<--"README.md"

```
